docs/default-firebase-data.json

```diff
@@ -817,6 +817,7 @@
       "shortBio": "Creator of Fuelio app. Fuelio was his hooby project started in 2012. Three years later the app wa...",
       "socials": [],
       "order": 0,
+      "rockstarSpeakerOrder": 0,
```

Comment:
No need to change in this file.
We are not using this one

Reply:
Its linked with test cases that is why I changed it